kaal-presentation/src/main/kotlin/cz/eman/kaal/presentation/fragment/KaalBindingScreen.kt

```diff
@@ -0,0 +1,70 @@
+@file:Suppress("unused")
+
+package cz.eman.kaal.presentation.fragment
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.lifecycle.LifecycleOwner
+
+/**
+ * Interface which brings binding functionality to fragment or activity.
+ *
+ * @param B Type of data binding class corresponding [layoutId]
+ * @author eMan a.s.
+ * @see 0.9.0
+ */
+interface KaalBindingScreen<B : ViewDataBinding> {
```

Comment:
Delegation should be used instead of inheritance -> this solution provides internal methods as public API

(It isn't blocking -> it should be rework in some other PR)